puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testRuby.groovy

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent { label 'fast' }
+
+    stages {
+        stage('Setup Git Repos') {
+            steps {
+                ghprb_git_checkout()
+            }
+        }
+        stage("Setup RVM") {
+            steps {
+                configureRVM(ruby)
```

Comment:
`ruby` is undef?

Reply:
Doesn't that come from the job parameters?

Reply:
ah no it comes from axis?

Reply:
That's what I expect